cmd/jimmsrv/service/service.go

```diff
@@ -398,6 +399,12 @@ func NewService(ctx context.Context, p Params) (*Service, error) {
 		JWTService:                 s.jimm.JWTService,
 	}
 
+	roleManager, err := role.NewRoleManager(&s.jimm.Database, s.jimm.OpenFGAClient)
```

Comment:
i propose we have a constructor of JIMM, so that i can do `jimm.New(params jimm.Params)`.
We often create an instance of JIMM in our tests, which would now be incomplete without us manually creating the roleManager instance and then setting it on JIMM.

Reply:
I think a different PR for this - and totally agree!

Reply:
can do in a follow-up.. please create a jira ticket so we don’t forget.. thanks